tests/ArraySchemaValidatorTest.php

```diff
@@ -50,22 +48,16 @@ public function testArrayOfStringsLegacyAndHackArrays(): void {
     ]);
     $validator->validate();
 
-    expect($validator->isValid())->toBeTrue();
-    $validated = $validator->getValidatedInput();
-    expect(C\count($validated['array_of_strings'] ?? vec[]))->toBeSame(4);
+    expect($validator->isValid())->toBeFalse();
   }
 
-  public function testUntypedArrayValid(): void {
+  public function testDictOfStringsInvalid(): void {
```

Comment:
i ran this with the `->coerceType` and ensured that the test failed since `->isValid()` returned `true`